README.md

```diff
@@ -96,13 +96,23 @@ import Client from 'shopify-buy/index.unoptimized.umd';
 ## Examples
 
 ### Initializing the Client
+If your store supports multiple languages, Storefront API can return translated resource types and fields. Learn more about [translating content](https://help.shopify.com/en/api/guides/multi-language/translating-content-api).
+
 ```javascript
 import Client from 'shopify-buy';
 
+// Initializing a client
 const client = Client.buildClient({
   domain: 'your-shop-name.myshopify.com',
   storefrontAccessToken: 'your-storefront-access-token'
 });
+
+// Initializing a client to return translated content
+const clientWithTranslatedContent = Client.buildClient({
+  domain: 'your-shop-name.myshopify.com',
+  storefrontAccessToken: 'your-storefront-access-token',
+  language: 'ja-JP',
```

Comment:
Could we remove the trailing comma here for consistency with the other code samples?